Makefile

```diff
@@ -38,53 +32,44 @@ undefine LINUX
 endif
 #####
 
-ifdef WIN32
-TCCREPO := https://github.com/vlang/tccbin_win
-VCFILE := v_win.c
-endif
-
 all: latest_vc latest_tcc
-ifdef WIN32
-	$(CC) $(CFLAGS) -std=c99 -municode -w -o v.exe $(TMPVC)/$(VCFILE) $(LDFLAGS)
-	./v.exe self
-else
 	$(CC) $(CFLAGS) -std=gnu11 -w -o v $(TMPVC)/$(VCFILE) $(LDFLAGS) -lm
 ifdef ANDROID
 	chmod 755 v
 endif
-	./v self
-ifndef ANDROID
-	$(MAKE) modules
-endif
-endif
+	@./v self
+
 ifdef V_ALWAYS_CLEAN_TMP
 	$(MAKE) clean_tmp
 endif
 	@echo "V has been successfully built"
 	@./v -version
 
 clean: clean_tmp
-	git clean -xf
+	@git clean -xf
```

Comment:
Hiding all these just makes debugging problems harder imo. This should be hidden when V stabilizes, not before :-| .